Source/WebKit/UIProcess/API/Cocoa/_WKWebPushAction.h

```diff
@@ -44,6 +44,7 @@ WK_EXTERN
 
 @property (nonatomic, readonly) NSNumber *version;
 @property (nonatomic, readonly) NSString *pushPartition;
```

Comment:
Should we deprecate `pushPartition`?

Reply:
Seems like we don't use it anywhere so I'll just remove it.